### ISSUE TITLE
One command deployment wizard

### ISSUE THREAD

**Issue by dhruvkb** [IMGIITRoorkee/omniport-docker#15]
Setting Omniport up still requires a lot of steps namely building containers, populating configuration YAML files and ENV files, keeping these files updated with time and with each other and so on.

It would make life much easier if there was a wizard, written in Bash or Python or something similarly ubiquitous, that can be cloned (or better `curl`ed), executed and then takes the user through the entire process of deployment step by step. The wizard must have checkpoints and must make full use of the rich text functionalities of the modern Linux terminal such as colour, highlighting and sound. It must also allow the user to change certain aspects of the deployment such as the ports and the domain name later on.

Something like the following would be ideal.

```console
$ sh -c "$(curl -fsSL https://raw.github.com/IMGIITRoorkee/omniport-docker/master/scripts/deploy/everything.sh)"
```

Long story short, the mission is to develop a TUI (not a CLI) for Omniport setup and management.
